e2e/run-tests.sh

```diff
@@ -1,18 +1,24 @@
 docker-compose up -d --build
+
 docker-compose run e2e npm run deploy
+
 docker-compose run -d bridge npm run watcher:signature-request
 docker-compose run -d bridge npm run watcher:collected-signatures
 docker-compose run -d bridge npm run watcher:affirmation-request
+
 docker-compose run -d bridge-erc npm run watcher:signature-request
 docker-compose run -d bridge-erc npm run watcher:collected-signatures
 docker-compose run -d bridge-erc npm run watcher:affirmation-request
+
 docker-compose run -d bridge-erc-native npm run watcher:signature-request
 docker-compose run -d bridge-erc-native npm run watcher:collected-signatures
 docker-compose run -d bridge-erc-native npm run watcher:affirmation-request
+
 docker-compose run -d bridge-erc-multiple npm run watcher:signature-request
 docker-compose run -d bridge-erc-multiple npm run watcher:collected-signatures
 docker-compose run -d bridge-erc-multiple npm run watcher:affirmation-request
-docker-compose run -d bridge-erc-multiple npm run watcher:bridge-deployed
+docker-compose run --name bridge_deployed -d bridge-erc-multiple npm run watcher:bridge-deployed
```

Comment:
if you've added the name,  add it it to all of them :)

Reply:
+1